src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -49,19 +49,15 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
 
   private webkitBrowsers: string[] = ['crios', 'fxios', 'safari', 'ios', 'ios-webview'];
 
-  private enableUnifiedPlanForChromiumBasedBrowsers: boolean;
   private recreateAudioContextIfNeeded: boolean;
 
   constructor({
-    enableUnifiedPlanForChromiumBasedBrowsers = false,
-
     // Temporarily disable this workaround while we work out the kinks.
```

Comment:
We can't just remove this right? If someone currently pass in this flag, it will throw an error.